ConfigSpace/read_and_write/pcs_new.py

```diff
@@ -169,28 +169,34 @@ def build_condition(condition):
     notequal_template = "%s | %s != %s"
     equal_template = "%s | %s == %s"
 
+    parent = condition.parent
+    if isinstance(parent, NumericalHyperparameter):
+        cond_values = str(condition.value)
+    else:
+        cond_values = [str(value) for value in condition.value]
```

Comment:
This looks like it is not correct. There could be an `InCondition` on a UniformIntegerHyperparameter and it would not be treated correctly.